build/attestations/provenance.md

```diff
@@ -0,0 +1,310 @@
+---
+title: Provenance attestations
+keywords: build, attestations, provenance, slsa
+description: Provenance build attestations
+---
+
+The provenance attestations include facts about the build process, including
+details such as:
+
+- Build timestamps
+- Build parameters and environment
+- Version control metadata
+- Source code details
+- Materials (files, scripts) consumed during the build
+
+Provenance attestations follow the
+[SLSA provenance schema, version 0.2](https://slsa.dev/provenance/v0.2#schema).
+
+## Create provenance attestations
+
+To create a provenance attestation, pass the `--attest type=provenance` option
+to the `docker buildx build` command:
+
+```console
+$ docker buildx build --tag <namespace>/<image>:<version> \
+    --attest type=provenance,mode=[min,max] .
```

Comment:
In the main page we use the `--provenance` flag exclusively - probably worth mentioning both the short-form and the long-form here.